src/lib/mock-observations/interfaces/index.ts

```diff
@@ -0,0 +1,11 @@
+export interface Generator {
+  /**
+   * initialise the generator with the given name and config.
+   */
+  initialise(name: String, config: {[key: string]: any}): void;
+  /**
+   * generate the next value, optionally based on historical values
+   */
+  next(historical: Object[] | undefined): Object;
+}
+export default Generator;
```

Comment:
can remove the `export default` statement, and only import with `{Generator}` from user code